election-day/skoved/election_day.go

```diff
@@ -0,0 +1,45 @@
+package electionday
+
+import "fmt"
+
+// NewVoteCounter returns a new vote counter with
+// a given number of initial votes.
+func NewVoteCounter(initialVotes int) *int {
+	voteCounter := initialVotes
```

Comment:
or `return &initialVotes`. You can also see the comment I left on that function on Andrea's PR.